pkg/crypto/api.go

```diff
@@ -41,8 +41,12 @@ type Crypto interface {
 	VerifyMAC(mac, data []byte, kh interface{}) error
 
 	// WrapKey will execute key wrapping of cek using apu, apv and recipient public key found in kh.
-	WrapKey(cek, apu, apv []byte, kh interface{}) (*composite.RecipientWrappedKey, error)
+	// 'opts' allows setting the option sender key handle using WithSenderKH() option. It allows ECDH-1PU key wrapping
+	// (aka Authcrypt). The absence of this option uses ECDH-ES key wrapping (aka Anoncrypt).
+	WrapKey(cek, apu, apv []byte, kh interface{}, opts ...WrapKeyOpts) (*composite.RecipientWrappedKey, error)
```

Comment:
The signature `WrapKey()` does not match that of webkms client implementations out there. The one I'm looking at accepts just the kekId, the unwrappedKey, and zcap stuff: https://github.com/digitalbazaar/webkms-client/blob/6225fe53cbef0979f9961371265e0513ca27df20/KmsClient.js#L199

Reply:
@llorllale this is not a translation of WebKMS, this is WrapKey in the Crypto api that works with LocalKMS and is aligned with ECDH/JWE. 

A layer on top of this (not in this framework of course) can be added to address capabilities, zcap, etc.

Reply:
Remote KMS client implementation will translate `kh` to a `keyId` that the remote KMS service will use to identify the key.